docs/DeveloperGuide.md

```diff
@@ -228,81 +229,88 @@ _{Explain here how the data archiving feature will be implemented}_
+  * Freelancers might want to know more about how they distribute their time and track the time spent on each project so that they can better understand their working style, and plan their time more efficiently.
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**:
 
+* Time tracking app
+  * Dashboard view for freelancer to get an overview of all projects
+  * Multiple groups to represent different categories to help in grouping of tasks such as freelance projects.
+  * Predefined groups that are commonly used for freelancers, such as projects and self-learning/improvement
+  * Timer to track the duration of a task or Indicate start time and end time for a task, so that the user can price their rates and charge clients more accurately.
+  * Reminder to start doing a task (only when application is open).
+  * Users can generate reports of time usage after a week/any specified duration to understand their work style better and optimise their time usage so that they can plan their time better.
+  * Users can also generate reports of their income over a period of time from various freelance projects
+  * Interesting/creative views for visualizing the reports/data.
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
+Priorities: High (must have) * `* * *`, Medium (nice to have) * `* *`, Low (unlikely to have) * `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​   | I want to …​                          | So that I can…​                        |
+| -------* | -------* | -----------------------------------* | ------------------------------------* |
```

Comment:
I think the asterisks here are causing Markdown to not format the table correctly.

Reply:
Fixed